### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by ryanmrubin**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -5,27 +5,22 @@ django-cas-provider
...
 
-To install, run the following command from this directory::
-
-    python setup.py install
-
-Or, put `cas_provider` somewhere on your Python path.
-
-If you want use CAS v.2 protocol or above, you must install `lxml` package to correct work.
+- Install this package.
+- If you want use CAS v.2 protocol or above, you must install the `lxml` package to correct work.
```
Comment (adamzap) [inline README.rst]: Grammar: `to correct work`

File: README.rst [hunk 2]
```diff
@@ -5,27 +5,22 @@ django-cas-provider
 OVERVIEW
 =========
 
-django-cas-provider is a provider for the `Central Authentication Service <http://jasig.org/cas>`_. It supports CAS version 1.0 and parts of CAS version 2.0 protocol. It allows remote services to authenticate users for the purposes of Single Sign-On (SSO). For example, a user logs into a CAS server
-(provided by django-cas-provider) and can then access other services (such as email, calendar, etc) without re-entering her password for each service. For more details, see the `CAS wiki <http://www.ja-sig.org/wiki/display/CAS/Home>`_ and `Single Sign-On on Wikipedia <http://en.wikipedia.org/wiki/Single_Sign_On>`_.
+django-cas-provider is a provider for the `my.crossway.org <https://my.crossway.org>`_. It allows remote Crossway services to authenticate users for the purposes of Single Sign-On (SSO). For example, a user logs into a CAS server
```
Comment (adamzap) [inline README.rst]: Grammar: `purposes` -> `purpose`?

File: README.rst [hunk 3]
```diff
@@ -5,27 +5,22 @@ django-cas-provider
 OVERVIEW
 =========
 
-django-cas-provider is a provider for the `Central Authentication Service <http://jasig.org/cas>`_. It supports CAS version 1.0 and parts of CAS version 2.0 protocol. It allows remote services to authenticate users for the purposes of Single Sign-On (SSO). For example, a user logs into a CAS server
-(provided by django-cas-provider) and can then access other services (such as email, calendar, etc) without re-entering her password for each service. For more details, see the `CAS wiki <http://www.ja-sig.org/wiki/display/CAS/Home>`_ and `Single Sign-On on Wikipedia <http://en.wikipedia.org/wiki/Single_Sign_On>`_.
+django-cas-provider is a provider for `my.crossway.org <https://my.crossway.org>`_. It allows remote Crossway services to authenticate users for the purposes of Single Sign-On (SSO). For example, a user logs into a CAS server
```
Comment (adamzap) [inline README.rst]: Grammar: `purposes` -> `purpose` ?

**Comment by adamzap** [comment]
Also, do we want to keep this as restructured text instead of markdown? Your call.

File: README.rst (hunk 3 above)
Comment (ryanmrubin) [inline README.rst]: SSO has many purposes, my friend. Some are benign, and others we dare not speak of.

File: README.rst (hunk 1 above)
Comment (ryanmrubin) [inline README.rst]: Fixed

File: README.rst (hunk 3 above)
Comment (ryanmrubin) [inline README.rst]: Fixed

**Comment by ryanmrubin** [comment]
I don't feel particular about md/rst. I'm not itching to change it.

**Pull request merged**